linux_os/guide/system/software/system-tools/package_python3-subscription-manager-rhsm_installed/rule.yml

```diff
@@ -18,6 +18,9 @@ severity: medium
 identifiers:
     cce@rhel8: 82243-7
 
+references:
+    srg: SRG-OS-000366-GPOS-00153
```

Comment:
From the package description "A small library for communicating with the REST interface of a Red Hat Unified Entitlement Platform. This interface is used for the management of system entitlements, certificates, and access to content."  Seems to be important to ensure that packages can be properly verified.  The mapping can be dropped if it is not needed.  The package is a dependency of the subscription-manager package.

Reply:
The python package is "A small library for communicating with the REST interface of a Red Hat Unified Entitlement Platform. This interface is used for the management of system
entitlements, certificates, and access to content."

It is installed by default and should probably not be removed since it handles entitlements, certificates, etc.

Reply:
Kept the mapping. Created https://github.com/ComplianceAsCode/content/issues/5017 to track conversation about including this rule in STIG profile.